### PR TITLE
white overlay as a component

### PR DESCRIPTION
**Pull request by ALFAROSO**

### CODE REVIEW

File: pages/report-hazard/style.css [hunk 1]
```diff
@@ -1,23 +1,3 @@
...
-  z-index: 10000;
-  clip-path: circle(150% at 100% 0);
-  transition: clip-path 1.9s ease-in-out;
-}
-.loader2 {
-  clip-path: circle(0% at 100% 100%);
-}
-/* end loading animation */
-
 #map {
```
Comment (whamester) [inline pages/report-hazard/style.css:1]: import ('../../assets/css/white-transition.css')

File: pages/report-hazard/index.html [hunk 2]
```diff
@@ -25,6 +25,8 @@
     <script src="../../modules/leaflet/leaflet.js"></script>
     <!-- Global Styles -->
     <link rel="stylesheet" href="../../assets/css/_global.css" />
+    <!-- White transition Styles -->
+    <link rel="stylesheet" href="../../assets/css/white-transition.css" />
```
Comment (whamester) [inline pages/report-hazard/index.html:29]: move this import to inside the style.css

File: pages/report-hazard/style.css (hunk 1 above)
Comment (ALFAROSO) [reply pages/report-hazard/style.css:3]: Done

File: pages/report-hazard/index.html (hunk 2 above)
Comment (ALFAROSO) [reply pages/report-hazard/index.html]: Done

**Review by whamester — APPROVED** [review]

**Pull request merged**